DROD/CharacterDialogWidget.cpp

```diff
@@ -4261,6 +4258,28 @@ const
 	return wstr;
 }
 
+WSTRING CCharacterDialogWidget::GetGotoDestinationName(
```

Comment:
If you decide to follow my [suggestion](https://github.com/CaravelGames/drod/pull/133#issuecomment-708294018) it'll likely be best to pass `WSTRING` directly to this function and rename it to `AppendDestrinationName` or something.